app/controllers/facility_reservations_controller.rb

```diff
@@ -111,7 +111,7 @@ def create
     @instrument =
       current_facility.instruments.find_by!(url_name: params[:instrument_id])
     @reservation = @instrument.admin_reservations.new
-    @reservation.assign_attributes(params[:admin_reservation])
+    @reservation.assign_attributes(params[:admin_reservation].merge(created_by: current_user.id))
```

Comment:
While we're at it, should we capture this for normal reservations as well? It would kind of be duplicating the `OrderDetail#created_by`, but it would make our data more consistent.

Reply:
I decided not to, as normal reservations go through ReservationCreator, which is rather more complicated. Seemed like more work than it was worth. Possibly there's a simpler way to handle capturing this though, idk if you have a suggestion.